jetson/nav/search/searchStateMachine.cpp

```diff
@@ -3,9 +3,12 @@
 #include "utilities.hpp"
 #include "stateMachine.hpp"
 #include "searchFromPathFile.hpp"
+#include "stateMachine.hpp"
```

Comment:
aren't all these added includes duplicates?

Reply:
oops missed that I'll fix